ajax/libs/zepto/package.json

```diff
@@ -20,6 +20,12 @@
       "files": [
         "**/*"
       ]
+    },
+    {
+      "basePath": "src",
+      "files": [
+        "!(zepto)*"
+      ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
Mistake?

Reply:
I just ran fixtool and this space is added by it... Have no idea what it was added. I push PR now and if the problem still exists in your opinion please help to fix. Thanks

Reply:
@sashberd Let's try to undo this change! A good chance to do practice.

Reply:
@extend1994 I removed this new line, but again it was added by fixtool of cdnjs tools. FYI